src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldWYSIWYG.class.php

```diff
@@ -710,6 +710,20 @@ protected function getEditorWidth()
         }
     }
 
+    /**
+     * {@inheritDoc}
+     */
+    public function ConvertPostDataToSQL()
```

Comment:
The issue mentions the problem for text fields, not just wysiwyg. So I would suggest moving the logic into the parent `TCMSFieldText`

Reply:
Done